framework/Models/Code.py

```diff
@@ -569,8 +570,12 @@ def evaluateSample(self, myInput, samplerType, kwargs):
         for f in fileList:
           os.remove(f)
 
+      returnDict.update(kwargs)
       returnValue = (kwargs['SampledVars'],returnDict)
-      return returnValue
+      exportDict = self.createExportDictionary(returnValue)
```

Comment:
why do we need this?

Reply:
This function is used to check the input data we retrieved from the code outputs, if the data is from from the raven sampled value, we will raise an warning, and we will replace the value with raven sampled value.